src/main/steps/applicant1/how-did-you-change-your-name/content.ts

```diff
@@ -14,7 +14,7 @@ const en = ({ isDivorce, required }) => ({
   anotherWayMoreDetails:
     'Provide details of when and how you changed your name. You will be asked to upload a photo or scan of the documents that prove you changed your name later in this application, or you can post them in. If you do not have any documents, explain why here.',
   errors: {
-    applicant1ChangedNameHow: {
+    applicant1NameChangedHow: {
```

Comment:
why the variable change?

Reply:
I think it's less confusing as it now matches the field in the case api